src/components/Card/Card.js

```diff
@@ -1,6 +1,45 @@
 import './Card.css'
 import createElement from '../../lib/createElement'
+import Button from '../Button/Button'
 
-export default function Card(name) {
-  return createElement('section', { className: 'Card' }, name)
+export default function Card(name, imageUrl) {
```

Comment:
If you consider my previous remark:
```suggestion
export default function Card({name, image}) {
```